### DISCUSSION TITLE
Sunbird-coKreat release 5.2.0

### DISCUSSION THREAD

**Discussion by vaivk369** [Sunbird-coKreat/Community#53] [General Discussions]
We are starting with 5.2.0 regression testing on staging environment. We will be referring to the microsites for the latest release notes and build tags for the same.

Reference of sunbird ED release-5.2.0 deployment:
https://github.com/orgs/Sunbird-Ed/discussions/390 

Knowlg: https://knowlg.sunbird.org/use/release-notes/release-5.3.0

@pallakartheekreddy @Krishnaj20 @surendrasinghs 
cc: @vinukumar-vs @mmanimegalai

**Comment by ashokreddy1208** [comment]
@vaivk369 Can you please let us know once deployment is done

**Reply by vaivk369** [reply]
Deployment for 5.2.0 is done on staging.

**Comment by ashokreddy1208** [comment]
Thanks @vaivk369 we will start 5.2 regression testing..

**Comment by vaivk369** [comment]
@santhosh-tg Please check if all the knowledge deployments are done as part of 5.2.0 release on coKreat staging?
